src/nuklear_tree.c

```diff
@@ -272,15 +272,15 @@ nk_tree_element_image_push_hashed_base(struct nk_context *ctx, enum nk_tree_type
     {nk_flags dummy = 0;
     struct nk_rect label;
     /* calculate size of the text and tooltip */
-    text_len = nk_strlen(title);
```

Comment:
Good catch, thanks. It seems `text_len` is not used anywhere then - could you also remove it (as well as the comment `/* calculate size of the text and tooltip */`)?